sky/skylet/providers/aws/config.py

```diff
@@ -235,7 +243,7 @@ def bootstrap_aws(config):
 
     # The head node needs to have an IAM role that allows it to create further
     # EC2 instances.
```

Comment:
Can we add: If `skypilot_iam_role` is True, we use our own IAM role for both head and workers.

Reply:
Added. Thanks!